lib/pagesRow.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+class CardRow extends StatelessWidget {
```

Comment:
```suggestion
class pagesRow extends StatelessWidget {
```

better if the name of the class has the name of the page.